frontend/packages/kubevirt-plugin/integration-tests/views/kubevirtDetailView.view.ts

```diff
@@ -0,0 +1,51 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { resourceRows } from '../../../../integration-tests/views/crud.view';
+
+export const createNic = $('#create-nic-btn');
+export const createDisk = $('#create-disk-btn');
+
+export const nicName = $('#nic-name');
+export const macAddress = $('#mac-address');
+export const networkTypeDropdownId = '#network-type';
+export const networkBindingId = '#binding';
+
+export const diskName = $('#disk-name');
+export const diskSize = $('#disk-size');
+export const diskStorageClassDropdownId = '#disk-storage-class';
+
+export const cancelBtn = $('button.kubevirt-cancel-accept-buttons.btn-default');
+export const applyBtn = $('button.kubevirt-cancel-accept-buttons.btn-primary');
+
+export const newResourceRowInput = $('[data-id=create-disk-row]')
```

Comment:
`newResourceRowInput` is also for `nic` , should it be less specific ?
```
export const newResourceRowInput = $('[data-id^=create-]')
```

Reply:
Fixed